baybe/parameters/enum.py

```diff
@@ -17,17 +19,39 @@ class CategoricalEncoding(ParameterEncoding):
     """Integer encoding."""
 
 
-class SubstanceEncoding(ParameterEncoding):
-    """Available encodings for substance parameters."""
-
-    MORDRED = "MORDRED"
-    """Encoding based on Mordred chemical descriptors."""
-
-    RDKIT = "RDKIT"
-    """Encoding based on RDKit chemical descriptors."""
-
-    MORGAN_FP = "MORGAN_FP"
-    """Encoding based on Morgan molecule fingerprints."""
+# TODO Ideally, this should be turned into a class that can:
+#  - return default when CHEM not installed
+#  - check if enum is fingerprint
+PARAM_SUFFIX_FINGERPRINT = "Fingerprint"
+
+if CHEM_INSTALLED:
+    import inspect
+
+    from baybe._optional.chem import BaseFingerprintTransformer, skfp_fingerprints
+
+    AVAILABLE_SKFP_FP = dict(
+        inspect.getmembers(
```

Comment:
@Hrovatin: I see you are playing around with getting the choices dynamically from `skfp`, which would be a really nice feature, as also discussed with @Scienfitz. I just want to raise awareness for one point, before you start spending hours on trying to get it running: I think it's more important to have a working auto-complete for the user than to have the enum auto-generated. If there is a way to achieve both – fine. But I'm starting to get the feeling that this is not possible, and in that case I think a hard-coded list is the better option. Thoughts?

Reply:
I made it literal now since it anyway caused typing errs without it